odoo/tools/populate.py

```diff
@@ -1,14 +1,6 @@
-import random
 from datetime import datetime
 from dateutil.relativedelta import relativedelta
-from odoo.tools import pycompat
-
-
-def Random(seed):
-    """ Return a random number generator object with the given seed. """
-    r = random.Random()
-    r.seed(seed, version=2)
-    return r
```

Comment:
probably best to ask @Xavier-Do just in case, but I agree

Reply:
I did on the original commit.

Reply:
After discussing this with xdo they specifically wanted the seed to be required, so I'll revert that bit (and better document it I guess).